packages/server/src/events/AutomationEmitter.ts

```diff
@@ -32,6 +32,8 @@ class AutomationEmitter implements ContextEmitter {
 
     if (chainAutomations === true) {
       return MAX_AUTOMATIONS_ALLOWED
+    } else if (env.isTest()) {
+      return 0
```

Comment:
This was the key to solving the reason `loop.spec.ts` has been timing out. We had lots of automations that triggered themselves, which was causing havoc in the background while other tests were running. This prevents that, unless chaining is explicitly enabled in the app metadata.